### PR TITLE
fix(docs): introduction

### PR DESCRIPTION
**Pull request by danforbes**
## Description

- In header navbar, libdocs link has been renamed from "Documentation" to "Packages"
- Added a "Using These Docs" section to Introduction
- Added a section about plugins to Introduction

### CODE REVIEW

File: docs/docs/guides/getting_started/introduction.md [hunk 1]
```diff
@@ -61,6 +58,31 @@ Web3.js is a modular collection of packages, each of which serves a specific nee
...
+
+Web3.js supports [plugins](/guides/web3_plugin_guide/), which are another way to encapsulate capabilities that support a specific need. There are plugins that exist to support native features, like those described by [EIPs](https://eips.ethereum.org/) as well as plugins that are designed to support specific smart contracts, middleware, or even other Ethereum-compatible networks. Visit the [Web3.js plugins homepage](https://web3js.org/plugins) to view a list of the most important Web3.js plugins, which includes:
+
+- [EIP-4337 (Account Abstraction) Plugin](https://www.npmjs.com/package/@chainsafe/web3-plugin-eip4337)
+
+- [EIP-4844 (Blob Transactions) Plugin](https://www.npmjs.com/package/web3-plugin-blob-tx)
+
+- [zkSync Plugin](https://www.npmjs.com/package/web3-plugin-zksync)
+
+## Features of Web3.js v4
```
Comment (jdevcs) [inline docs/docs/guides/getting_started/introduction.md]: I think we should keep this overview under introduction section

Comment (jdevcs) [reply docs/docs/guides/getting_started/introduction.md]: but at same time if we want to highlight "Using These Docs" section , what do you think about using https://docusaurus.io/docs/markdown-features/admonitions ?

Comment (danforbes) [reply docs/docs/guides/getting_started/introduction.md]: Oh, I just moved the section back to its original location 😅 Are you suggesting that "Features of Web3.js v4" get moved back to its original location and then we use an admonition for the "Using These Docs" section? I do not necessarily like the idea of using an admonition for "Using These Docs" as I think it's too much text to go into an admonition...

**Review by jdevcs — APPROVED** [review]
looks good, just added few updates ideas, Thanks @danforbes

**Pull request merged**